config/mastodon.yml

```diff
@@ -5,7 +5,7 @@ shared:
   software_update_url: <%= ENV.fetch('UPDATE_CHECK_URL', 'https://api.joinmastodon.org/update-check') %>
   source:
     base_url: <%= ENV.fetch('SOURCE_BASE_URL', nil) %>
-    repository: <%= ENV.fetch('GITHUB_REPOSITORY', 'mastodon/mastodon') %>
```

Comment:
I see the comments about why this changed, which is fine. If we did want to keep the defaults in here (seems *slightly better* to me, but I don't feel strongly), I think something like:

`repository: <%= ENV.fetch('GITHUB_REPOSITORY', nil).presence || 'mastodon/mastodon' %>`

would work? We'd get a value when set, but when either unset or set to empty string, we'd use our default.